test/models/user_test.rb

```diff
@@ -4,57 +4,49 @@
 
 class UserTest < ActiveSupport::TestCase
   def setup
-    @user = users(:one)
+    @user = users(:john)
   end
 
   test 'following? should return true if user is being followed' do
-    user_to_follow = users(:two)
+    user_to_follow = users(:mike)
     @user.follow(user_to_follow)
 
     assert @user.following?(user_to_follow)
```

Comment:
```suggestion
    @john.follow(mike)

    assert @john.following?(mike)
```

みたいになってる方が僕は読みやすいかな、と思います。